x-pack/test_serverless/api_integration/test_suites/common/reporting/generate_csv_discover.ts

```diff
@@ -84,6 +84,7 @@ export default function ({ getService }: FtrProviderContext) {
       });
 
       after(async () => {
+        await reportingAPI.deleteAllReports();
```

Comment:
With this, it seems the tests are indeed fixed for MKI runs (gave it a try against an observability project). With that, I think we can remove the `failsOnMKI` tag and the corresponding comment (L71-72).

Reply:
Good call thank you - removed in [c217b59](https://github.com/elastic/kibana/pull/180479/commits/c217b599b83e9425678fa3062e29ca7fba93f4dc)

Reply:
@rshen91 I see that the comment has been removed. However, the `this.tags(['failsOnMKI']);` line is still there, which should also be removed in order to re-enable the suite for MKI runs.

Reply:
d'oh! My mistake removed 👌🏻